app/Console/Commands/VideoData.php

```diff
@@ -67,7 +66,7 @@ public function handle()
 			)
 		);
 
-		$photos = Photo::where('type', 'like', 'video/%')
+		$photos = Photo::whereIn('type', $this->photoFunctions->getValidVideoTypes())
```

Comment:
Less permissive, but IIRC more in line with how we handle this elsewhere, so makes sense.

Reply:
Actually, check: https://github.com/LycheeOrg/Lychee/blob/2b0664a09dea2ca9533c1879bffad7c97b36c519/app/ModelFunctions/PhotoFunctions.php#L51-L62
As you can see, one video type (MPG) is sometimes recognized as `image/x-tga`, so this change actually makes it more inclusive...